Startup.cs

```diff
@@ -27,8 +28,10 @@ public void ConfigureServices(IServiceCollection services)
             services.AddHttpClient<swaggerClient>(client =>
             {
                 var authToken = Configuration["AuthToken"];
+                client.DefaultRequestHeaders.Add(HeaderNames.UserAgent, "Kevin's cool thing.");
```

Comment:
I noticed that the darc commandline client is sending a user agent header. 

This is a nice thing to do when there's API versioning at play because the owner of the app can know who's still using the older API. It's just a friendly thing, and doesn't seem required.